ios/fastlane/Constants/Constants.rb

```diff
@@ -67,21 +67,21 @@ def self.SCHEME_NAME_PRODUCTION
 
   # an staging product name
   def self.PRODUCT_NAME_STAGING
-    'Flutter Template Staging'
+    'Runner'
```

Comment:
Any reason it's called `Runner`? 🙇

Reply:
@Tuubz This is the default name of the project but if you install the app from testflight you will see name `SurveyProduction` or `SurveyStaging`